README.md

```diff
@@ -4,9 +4,17 @@ Website for [EFF's Certbot](https://certbot.eff.org/) project. Uses Jekyll for s
 
 [![Build Status](https://travis-ci.org/certbot/website.svg?branch=master)](https://travis-ci.org/certbot/website)
 
-## Getting Started
+## Development
 
-### Install
+### With Docker
+
+`docker-compose up`
```

Comment:
The docker build fails if you don't have the _docs submodule cloned. Those steps (4/5) are now under "Without Docker" which is a little confusing.